docker-compose.yml

```diff
@@ -89,3 +89,13 @@ cli:
 #     - web
 #   ports:
 #     - "4444:4444"
+
+# To use Solr, uncomment the service definition below and link the web service to the solr service.
+# solr:
+#   hostname: solr
+#   image: blinkreaction/drupal-solr:3.x
+#   ports:
+#     - "8983:8983"
+#   volumes:
```

Comment:
volumes not needed
 - /var/lib/solr/data is defined in the base image 
 - I moved the conf files into the solr:3.x image - https://github.com/blinkreaction/docker-drupal-solr/tree/master/solr3.x/solr/conf
- don't forge the new line at EOF :)